bi_view_editor/models/bve_view.py

```diff
@@ -98,6 +98,11 @@ def _inverse_serialized_data(self):
         compute='_compute_users',
         store=True)
     query = fields.Text(compute='_compute_sql_query')
+    over_condition = fields.Text(
+        string="Over condition",
```

Comment:
You can remove this line `string="Over condition",` since Odoo creates the field name automatically.

Reply:
Thanks for the review!
Yes I know that the `string` is automatically filled but I usually prefer to keep it explicit.

Reply:
As per https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#fields, the string should be removed. It's also useful to not specify any string in some cases. For example, in case someone wants to manually set a different string directly in the field definition, this won't conflict when the module is updated.

Reply:
Removed